data/sql/updates/pending_db_world/haggle.sql

```diff
@@ -0,0 +1,124 @@
+(14041, 0, 5, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP20 reached - Say emote 2'),
+(14041, 0, 6, 7, 40, 0, 100, 0, 22, 0, 0, 0, 0, 0, 54, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP22 reached - Pause waypoint'),
+(14041, 0, 7, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP22 reached - Say emote 2'),
+(14041, 0, 8, 9, 40, 0, 100, 0, 27, 0, 0, 0, 0, 0, 54, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP27 reached - Pause waypoint'),
+(14041, 0, 9, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 5, 69, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP27 reached - Do emote OS crafting'),
+(14041, 0, 10, 11, 40, 0, 100, 0, 30, 0, 0, 0, 0, 0, 54, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP30 reached - Pause waypoint'),
+(14041, 0, 11, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 5, 69, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP30 reached - Do emote OS crafting'),
+(14041, 0, 12, 0, 40, 0, 100, 0, 32, 0, 0, 0, 0, 0, 80, 1404104, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP32 reached - Run Actionlist'),
+(14041, 0, 13, 0, 40, 0, 100, 0, 34, 0, 0, 0, 0, 0, 80, 1404103, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP34 reached - Run Actionlist'),
+(14041, 0, 14, 15, 40, 0, 100, 0, 45, 0, 0, 0, 0, 0, 54, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP45 reached - Pause waypoint'),
+(14041, 0, 15, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 5, 69, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP45 reached - Do emote OS crafting'),
+(14041, 0, 16, 17, 40, 0, 100, 0, 48, 0, 0, 0, 0, 0, 54, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP48 reached - Pause waypoint'),
+(14041, 0, 17, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 5, 69, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP48 reached - Do emote OS crafting'),
+(14041, 0, 18, 19, 40, 0, 100, 0, 51, 0, 0, 0, 0, 0, 54, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP51 reached - Pause waypoint'),
+(14041, 0, 19, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 5, 69, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP51 reached - Do emote OS crafting'),
+(14041, 0, 20, 21, 40, 0, 100, 0, 55, 0, 0, 0, 0, 0, 54, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP55 reached - Pause waypoint'),
+(14041, 0, 21, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 5, 69, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP55 reached - Do emote OS crafting'),
+(14041, 0, 22, 23, 40, 0, 100, 0, 59, 0, 0, 0, 0, 0, 54, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP59 reached - Pause waypoint'),
+(14041, 0, 23, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 5, 69, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Haggle - On WP59 reached - Do emote OS crafting'),
```

Comment:
Are these emotes needed? Since the creature_text already has it.
Also these actions could be halved with an Actionlist shared between these points

Reply:
He seemed to do the emote but not the text in these cases.